lib/firmata.js

```diff
@@ -1689,6 +1718,333 @@ Board.prototype.pingRead = function(opts, callback) {
   this.once("ping-read-" + pin, callback);
 };
 
+/**
+ * Stepper functions to support version 2 of ConfigurableFirmata's asynchronous control of stepper motors
+ * https://github.com/soundanalogous/ConfigurableFirmata
+ */
+
+/**
+ * Asks the arduino to configure a stepper motor with the given config to allow asynchronous control of the stepper
+ * @param {object} opts Options:
+ *    {number} deviceNum: Device number for the stepper (range 0-9)
+ *    {number} type: One of this.STEPPER.TYPE.*
+ *    {number} stepType: One of this.STEPPER.STEPTYPE.*
+ *    {number} stepPin: Only used if STEPPER.TYPE.DRIVER
+ *    {number} directionPin: Only used if STEPPER.TYPE.DRIVER
+ *    {number} motorPin1: motor pin 1
+ *    {number} motorPin2:  motor pin 2
+ *    {number} [motorPin3]: Only required if type == this.STEPPER.TYPE.THREE_WIRE || this.STEPPER.TYPE.FOUR_WIRE
+ *    {number} [motorPin4]: Only required if type == this.STEPPER.TYPE.FOUR_WIRE
+ *    {number} [enablePin]: Enable pin
+ *    {array} [invertPins]: Array of pins to invert
+ */
+
+Board.prototype.accelStepperConfig = function(opts) {
+
+  var interface, pinsToInvert = 0x00;
```

Comment:
`interface` is a reserved keyword in strict mode code. Ideally we want this code to be strict mode safe and es module safe (which is implicit strict mode). Since I've taken so long to review, I will make the fixes for you